tests/plugins/inventory/netbox/2.8.9/mocked/platforms_conn_opts.json

```diff
@@ -0,0 +1,30 @@
+{
+  "count": 2,
+  "next": null,
+  "previous": null,
+  "results": [
+    {
+      "id": 1,
+      "name": "junos",
+      "slug": "junos",
+      "manufacturer": {
+        "id": 1,
+        "url": "http://localhost:8080/api/dcim/manufacturers/1/",
+        "name": "juniper",
+        "slug": "juniper"
+      },
+      "napalm_driver": "junos",
+      "napalm_args": {
```

Comment:
shouldn't this be the options for a connection plugin?
```
"netmiko": {
  "extras": {
....
```

Reply:
Yep. Fixed